scripts/create_workspace.py

```diff
@@ -0,0 +1,26 @@
+# this script is called by AzureML notebooks to create workspace config file for the first run 
```

Comment:
in all the files we have to add the copyright:

```
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.
```

Reply:
Added now